packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.ct.tsx

```diff
@@ -0,0 +1,42 @@
+import { expect, test } from "../../playwright-axe";
+import OnyxRadioButton from "./OnyxRadioButton.vue";
+
+test("should display correctly", async ({ mount, makeAxeBuilder, page }) => {
+  // ARRANGE
+  await mount(
+    <OnyxRadioButton id="my-id" label="radio label" name="radio form" value="radio-value" />,
+  );
+
+  // ASSERT
+  await expect(page.getByRole("radio")).toBeAttached();
+  await expect(page.getByLabel("radio label")).toBeAttached();
+  await expect(page.getByText("radio label")).toBeAttached();
+
+  // ACT
+  const accessibilityScanResults = await makeAxeBuilder().analyze();
+
+  // ASSERT
+  expect(accessibilityScanResults.violations).toEqual([]);
+});
+
+test("should display correctly when selected", async ({ mount, makeAxeBuilder, page }) => {
+  // ARRANGE
+  await mount(
+    <OnyxRadioButton
+      id="my-id"
+      label="radio label"
+      name="radio form"
+      value="radio-value"
+      selected
+    />,
+  );
+
+  // ASSERT
+  await expect(page.getByRole("radio")).toBeChecked();
+
+  // ACT
+  const accessibilityScanResults = await makeAxeBuilder().analyze();
+
+  // ASSERT
+  expect(accessibilityScanResults.violations).toEqual([]);
+});
```

Comment:
disabled is a feature that already works. should there be a test for it or no, because it's a html standard behavior?

Reply:
done